tutorial/tutorial.md

```diff
@@ -127,26 +102,21 @@ qiime qsip2 create-qsip-data \
   --o-qsip-data qsip-data.qza
 ```
 
-Each of the column names has a default value that can be seen by running
-`qiime qsip2 create-qsip-data --help` and reading the resulting command line
-output. If a column in your metadata is already named with this default then
-you do not need to provide that argument. For example, if your
-**source material id** column is alread called `source_mat_id` you can drop
-the `--p-source-mat-id` argument from the command. Note that in this example
-we are dropping the `--p-isotoplog-column` argument from the command because
-our isotoplog column is already named with the default. If you isotopolog
-column has a different name, you should add this argument with that name.
+(Note: When working with your own data, if you have a single metadata file, the software treats this a sample-level metadata, and you would simply not provide the `--p-source-metadata` argument.)
```

Comment:
This is nitpick-y, but I think the single file/double file business is one of the more confusing parts of the workflow. If I were reading through this, I might glaze over anything parenthetical--do you mind if we leave this in the main paragraph?

Reply:
Good thought. If/when we convert this to a Jupyter Book, we can put this in an admonition block to make it stand out. For now, I put the text about running on your own data in one single paragraph, and bold-faced some text to make it stand out: 
<img width="552" alt="Screenshot 2024-09-19 at 5 57 59 AM" src="https://github.com/user-attachments/assets/0050f65d-4249-41f2-b2b9-2280abb356e2">

How's that?

Reply:
Sounds great.